.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+    Tag your PR title with the components that it touches along with
+    the type of change (feat, fix, refactor)
+    E.g. "fix: fix activity 1 button placement" or "feat: add favicon and make footer thinner"
+-->
+
+## Summary
+
+Closes #<!-- issue number -->
+
+<!-- Enumerate changes you made and why you made them -->
+
+## Additional Information
```

Comment:
Should we maybe have a test plan (maybe instead of additional information)? Just to make sure that people actually manually test the code?

Reply:
hm ok, so far in the example you provided in the issue, there is this: 
`## Test Plan
<!--
    How did you test your change? How do you know it works?
    Add supporting screenshots, terminal pastes, etc. as necessary
-->`
I think this is good but could also be a bit vague; could you give some examples for the kind of testing people could do (and then we could list them on the template)? off the top of my head i can think of testing accessibility stuff like font changes and window resizing / trying it on different interface sizes, but idk how in-depth we want testing to be in terms of like code logic

Reply:
Ohhh testing as in making sure that the code actually works. So some examples could be screenshot of the feature in play, or any other way they manually tested it